src/plugins/dashboard/public/application/dashboard_app.tsx

```diff
@@ -303,7 +303,7 @@ export function DashboardApp({
   }, [data.search.session]);
 
   return (
-    <>
+    <div className="dshAppContainer">
```

Comment:
It doesn't look like this class has any styles attached to it or targeted specifically in any way. So if it's just the `<div>` that's absolutely needed, then be sure to use the `APP_WRAPPER_CLASS` as the `className`.

Reply:
https://github.com/elastic/kibana/blob/9b2d0c335441f2e32c6aea77f6c10d5e5d11c108/src/core/utils/app_wrapper_class.ts

Reply:
Looks like it matches CSS that is injected when the styles are updated for print layout: https://github.com/elastic/kibana/blob/master/x-pack/plugins/reporting/server/lib/layouts/print.css#L99-L122

It looks like it has been used for this since the beginning of the print layout feature, and my guess is it has the funny name so that it doesn't affect anything else.

A more maintainable solution will be to have a different view under the app that renders with the layout that is expected for the report, and send Reporting the URL to that view of the data.

Reply:
Ok, so then based on that usage, it's being used as a parent selector but applying styles to child panels. You don't have to use this specific `.dshAppContainer` element/selector. There's one just one layer down that will still encompass the children they are selecting. `.dashboardViewport` on line 332. So you can change the selector in the `print.css` to that and keep the extra div out.